### ISSUE TITLE
Passing the ssh password via environment variable not working.

### ISSUE THREAD

**Issue by skakee** [cloudflare/gortr#26]
According to help:
```
[cloudflare@plebe ~]$ gortr -help
Usage of gortr:
...
  -ssh.auth.password string
        SSH password (if blank, will use envvar GORTR_SSH_PASSWORD)
...
```

Specifying the password with `-ssh.auth.password` option works:
```
[cloudflare@plebe ~]$ /opt/cloudflare/go/bin/gortr -metrics.addr :8281 -ssh.bind :8282 -ssh.key /opt/cloudflare/private.pem -ssh.method.password=true -ssh.auth.user rpki -ssh.auth.password SOME_PASSWORD -verify=false -cache http://localhost:8080/output.json -refresh=120 -bind ""
INFO[0000] Enabling ssh with the following authentications: password=true, key=false
INFO[0001] New update (99057 uniques, 99057 total prefixes). 0 bytes. Updating sha256 hash  -> fb71bdd9a3adfb5caa2729891e739b2647bae9a0b5ca56d4d6c24d48614857c4
INFO[0001] Updated added, new serial 1
INFO[0002] Accepted ssh connection from 127.0.0.1:59326 (3/0)
INFO[0002] Connected (ssh-password): rpki/127.0.0.1:59326
```

But setting the environment variable and setting `-password` to blank, ie `""` does not:
```
[cloudflare@plebe ~]$ export GORTR_SSH_PASSWORD=SOME_PASSWORD
[cloudflare@plebe ~]$ /opt/cloudflare/go/bin/gortr -metrics.addr :8281 -ssh.bind :8282 -ssh.key /opt/cloudflare/private.pem -ssh.method.password=true -ssh.auth.user rpki -ssh.auth.password "" -verify=false -cache http://localhost:8080/output.json -refresh=120 -bind ""
INFO[0000] Enabling ssh with the following authentications: password=true, key=false
INFO[0000] New update (99057 uniques, 99057 total prefixes). 0 bytes. Updating sha256 hash  -> fb71bdd9a3adfb5caa2729891e739b2647bae9a0b5ca56d4d6c24d48614857c4
INFO[0000] Updated added, new serial 1
INFO[0002] Accepted ssh connection from 127.0.0.1:33434 (1/0)
INFO[0002] Connected (ssh-password): rpki/127.0.0.1:33434
WARN[0002] Wrong user or password for rpki/127.0.0.1:33434. Disconnecting.
ERRO[0002] Error with ssh client 127.0.0.1:33434: [ssh: no auth passed yet, Wrong user or password]
```

Omitting `-ssh.auth.password` also does not work.

**Comment by lspgn** [comment]
Hi @skakee,
Looks like there is a typo in the code: https://github.com/cloudflare/gortr/blob/master/cmd/gortr/gortr.go#L35. It is `RTR_SSH_PASSWORD` instead of `GORTR_SSH_PASSWORD`

Will fix.

**Comment by lspgn** [comment]
Should be fixed in the latest release (v0.11.4), please confirm :)
